python/adjoint/objective.py

```diff
@@ -82,19 +82,25 @@ def place_adjoint_source(self,dJ):
         
         # an ugly way to calcuate the scaled dtft of the forward source
         y = np.array([self.time_src.swigobj.current(t,dt) for t in np.arange(0,T,dt)]) # time domain signal
-        signal_dtft = np.matmul(np.exp(1j*2*np.pi*self.frequencies[:,np.newaxis]*np.arange(y.size)*dt), y)
-        source_amp = np.abs([self.time_src.fourier_transform(f) for f in self.frequencies]) * np.exp(2*1j*np.angle(signal_dtft)) # note the fudge factor of 2 in the phase...
+        signal_dtft = np.matmul(np.exp(1j*2*np.pi*self.frequencies[:,np.newaxis]*np.arange(y.size)*dt), y) # dtft
+        fwd_src_phase = np.exp(1.0*1j*np.angle(signal_dtft)) # phase component of dtft
+        fwd_src_amp = np.abs([self.time_src.fourier_transform(f) for f in self.frequencies]) # amplitude of source
```

Comment:
Is this a DTFT?

Reply:
This is out of date...let me push an update

Reply:
Why not just use `np.abs(signal_dtft)` with the appropriate scale factor (1/2π or sqrt(1/2π))?

Reply:
Yes, it's a dtft.

Reply:
What is the difference between `signal_dtft` and `self.time_src.fourier_transform(f)`, then? Why are we computing both?

Reply:
There's an "amplitude" discrepancy when I manually compute the DTFT. For some reason, the actual amplitude of the source matches the continuous FT approximation, rather than the DTFT computation...